frontend/package.json

```diff
@@ -11,15 +11,17 @@
   "private": true,
   "dependencies": {
     "@angular/animations": "14.1.0",
+    "@angular/cdk": "^14.1.0",
     "@angular/common": "14.1.0",
     "@angular/compiler": "14.1.0",
     "@angular/core": "14.1.0",
     "@angular/forms": "14.1.0",
+    "@angular/material": "^14.1.0",
```

Comment:
material is overkill for such a simple component, refactor using bootstrap or plain html, it does not need to be pretty, the functionality is what is importan if you have to use  <ul> that is enough for this use case.

Reply:
Angular material removed